core/fokia/utils.py

```diff
@@ -28,12 +34,14 @@ def patch_certs(cert_path=None):
                     get_default_verify_paths().openssl_cafile
```

Comment:
This exists in Ubuntu by default and returns '/usr/lib/ssl/cert.pem'. However the file does not exist.